recipes/mss/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "1.2.0" %}
+
+package:
+    name: mss
+    version:  {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    skip: True  # [py3k]
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+    build:
+        - python
+        # mss import pyproj from basemap and that does not exist in 1.0.8
+        - basemap 1.0.7
+        - chameleon
+        - execnet
+        - lxml
+        - netcdf4
```

Comment:
Hmm, yes. But if I do first install the meta package and afterwards upgrade by 
conda install -c conda-forge netCDF4 it doesn't. 

my example:

```
rm -rf miniconda2
./Miniconda2-latest-Linux-x86_64.sh
conda install conda-build
conda create -n TEST python=2
source activate TEST

cd conda_test

conda build . # your yaml
conda install --use-local mss

```
```
 conda list | grep netcdf
libnetcdf                 4.3.3.1                       3  
netcdf4                   1.2.2               np111py27_0 
```

```
conda install -c conda-forge netcdf4
python -c "import netCDF4; print(netCDF4.__version__)"
```

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/mss/miniconda2/envs/TEST/lib/python2.7/site-packages/netCDF4/__init__.py", line 3, in <module>
    from ._netCDF4 import *
ImportError: libmfhdf.so.0: cannot open shared object file: No such file or directory

```

I was not yet aware that it happens because of the upgrade. So we are a bit closer.